new-pipeline/dist.html

```diff
@@ -23,185 +23,266 @@
-                                <div id="trim-option">
-                                    <b>Trim small buckets:</b> hide buckets on both sides that don't have enough samples:
-                                    <label><input name="trim-toggle" type="radio" value="0"> Don't Trim Buckets</label>
-                                    <label><input name="trim-toggle" type="radio" value="1"> Trim Buckets</label>
+                </div>
+                <div class="row">
+                    <div class="col-md-12">
+                        <div class="panel panel-default">
+                            <div class="panel-heading" role="tab">
+                                <a class="panel-title" data-toggle="collapse" id="advanced-settings-toggle" href="#advanced-settings">Advanced Settings<span></span></a>
+                            </div>
+                            <div id="advanced-settings" class="panel-collapse collapse">
+                                <div class="panel-body">
+                                    <form class="form-horizontal">
+                                        <div>
+                                            Evolution variable: show evolution over build date or submission date:
+                                            <label><input name="build-time-toggle" type="radio" value="0"> Build Date</label>
+                                            <label><input name="build-time-toggle" type="radio" value="1"> Submission Date</label>
+                                        </div>
+                                        <div>
+                                            Sanitize data: hide data points that don't have enough submissions:
+                                            <label><input name="sanitize-toggle" type="radio" value="0"> Don't Sanitize</label>
+                                            <label><input name="sanitize-toggle" type="radio" value="1"> Sanitize</label>
+                                        </div>
+                                    </form>
                                 </div>
-                                <span id="date-range-controls" class="form-group" style="width: 230px; display: inline-block; margin: 0">
-                                    <span class="input-prepend input-group">
-                                        <span class="add-on input-group-addon"><i class="glyphicon glyphicon-calendar fa fa-calendar"></i></span>
-                                        <input type="text" name="date-range" style="width: 100%" id="date-range" class="form-control date-range">
-                                    </span>
-                                </span>
-                                <span id="range-bar" class="pull-right"></span>
-                            </form>
+                            </div>
                         </div>
                     </div>
                 </div>
+                <div class="busy-indicator">
```

Comment:
The busy indicator needs to be in the correct stacking context to be displayed without padding on the left. I don't think it can be moved in here.